gbpservice/nfp/bin/startup/redhat/nfp_proxy.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=One Convergence's NFP Proxy
+After=syslog.target network.target
+
+[Service]
+ExecStart=source /opt/stack/gbp/devstack/lib/nfp;namespace_delete /home/stack/devstack;namespace_create /home/stack/devstack 11.0.0.3
```

Comment:
Ahmed, could you please check Pritam's question on the location of namespace. Will this work?